cloud-provider-azure-1.31.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: cloud-provider-azure-1.31
+  version: 1.31.1
+  epoch: 0
+  description: Cloud provider for Azure
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-sigs/cloud-provider-azure
+      tag: v${{package.version}}
+      expected-commit: 301bb5195472aa76bb36e24726c197ccbb19cfca
+
+subpackages:
+  - name: "azure-cloud-controller-manager-1.31"
```

Comment:
I think we should consider changing the name to something where we don't manually hardcode the version to 1.31. This is because when automation bumps the PR, it'll not change this name. We can go with `${{package.name}}-cloud-controller-manager` name and add a provides block similar to what was done here via this patch (https://github.com/wolfi-dev/os/pull/33097)

Reply:
Thanks for the review @kranurag7, from our Slack conversation I think I will go back to single package.